test/functional/test-srcset.js

```diff
@@ -78,6 +78,54 @@ describe('Srcset parseSrcset', () => {
     expect(res.sources_[2].dpr).to.equal(undefined);
   });
 
+  it('should accept commas in URLs', () => {
+    var res = parseSrcset(' \n image,1 100w\n , \n image,2 50w \n');
```

Comment:
Add more examples here with some kind of test helper function to make that only a few lines.